src/cli/java/org/commcare/util/screen/SessionUtils.java

```diff
@@ -138,7 +138,7 @@ public int doPostRequest(
             PrintStream printStream
     ) throws IOException {
         String url = buildUrl(syncPost.getUrl().toString());
-        Multimap<String, String> params = syncPost.getEvaluatedParams(session.getEvaluationContext());
+        Multimap<String, String> params = syncPost.getEvaluatedParams(session.getEvaluationContext(), true);
```

Comment:
I'm not 100% sure whether this should be passing true or false.

Reply:
Based on the usage here: https://github.com/dimagi/formplayer/blob/671ea55f7e3780e5b4e0dc2395d96812ce3af18f/src/main/java/org/commcare/formplayer/screens/FormplayerSyncScreen.java#L37-L38 I would say it should be `false`.

However, looking back in the history I'm not 100% sure what it should be. We [introduced the `exclude` attribute on post data elements](https://github.com/dimagi/commcare-core/commit/0a6be291a021c59941a13350437b7d5dc6bca6f2) and for a simple value data it's clear that if `exclude` evaluates to `true` then the entire param should be excluded. For data elements have a nodeset, the `exclude` expression applies to [each individual item in the list](https://github.com/dimagi/commcare-core/blob/3265df37fce0bcbecc40ab6c8c1bc4b72a8504ef/src/main/java/org/commcare/suite/model/ListQueryData.java#L64-L66). So the question is, if the list is empty, should the param still be included or not?

For now I'll change this param should be `false`.